### DISCUSSION TITLE
태그도 카테고리로 묶을 수 있으면 좋겠습니다!

### DISCUSSION THREAD

**Discussion by simyat** [morethanmin/morethan-log#169] [Ideas]
게시글을 카테고리로 묶을 수 있는 것처럼 태그도 카테고리로 묶을 수 있게 만들어주세요!!
좋은 서비스 늘 감사합니다 ㅎㅎ

**Comment by morethanmin** [comment]
안녕하세요! 혹시 좀 더 구체적으로 설명해주실 수 있으실까요? 예시가 있으면 좀더 이해하기 좋을 것 같습니다. ㅎㅎ

**Reply by kjmn1105** [reply]
저도 이런 식으로 그룹핑이 가능하면 더 보기 편할 것 같습니다!

🏷️ Tags

1번 태그  (12)
⎿ 1-1.      (2)
⎿ 1-2.     (3)
⎿ 1-3.     (7)

2번 태그  (5)
⎿ 2-1.    (1)
⎿ 2-2.   (4)

**Reply by morethanmin** [reply]
@simyat @jhk0530 

아하 넵 확인했습니다. ui 개선 작업할 때 참고해서 작업하겠습니다! 감사합니다. ㅎㅎ
